### DISCUSSION TITLE
Allow embedding authentik

### DISCUSSION THREAD

**Discussion by svanimpelen** [goauthentik/authentik#4345] [Q&A]
Currently using Authentik for a few docker containers. I have some of those services in a Web card within Home assistant. On desktop this works fine. In the mobile app I get an error and on the mobile firefox browser i see the following message:

"**Nightly cant open this page**

To protect your ssecurity, auth.mydomain.com will not allow Nightly to display the page if another site has embedded it. To see this page, you need to open it in a new windows."

Does anyone know if there is a setting that allows embedding?

**Comment by Simie** [comment]
I had the exact same problem with home assistant iframes and solved it using a traefik middlware:

```
  labels:
      - "traefik.http.routers.authentik-rtr.middlewares=allow-framing"
      - "traefik.http.middlewares.allow-framing.headers.customresponseheaders.Content-Security-Policy=frame-ancestors 'self' https://<my domain> https://*.<my domain>"
```
